examples/external_bvh_raycasting.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - materials - cube refraction [Lucy]</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+		<style>
+			body {
+				color: #333;
+			}
+
+			a {
+				color: #E91E63;
+				text-decoration: underline;
+			}
+		</style>
+	</head>
+
+	<body>
+		<div id="info">
+			<a href="https://github.com/gkjohnson/three-mesh-bvh">three-mesh-bvh</a> accelerated raycasting demo<br/>
+			See <a href="https://github.com/gkjohnson/three-mesh-bvh">main project repository</a> for more information and examples on high performance spatial queries.
+		</div>
+
+		<!-- Import maps polyfill -->
+		<!-- Remove this when import maps will be widely supported -->
+		<script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js"></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three-mesh-bvh": "https://unpkg.com/three-mesh-bvh/build/index.module.js"
```

Comment:
Isn't it more safe to use a fixed version here? What happens if there is a breaking change in `three-mesh-bvh` which affects the example?

Reply:
Yeah I was thinking about what the best thing to do would be. I changed this to use a non breaking version using `^`, instead.